modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc

```diff
@@ -567,8 +681,8 @@ class NDB_Form_Data_Team_Helper extends NDB_Form
      * Returns the percent completed for the given instrument
      *
      * @param unknown_type $instrument  the value of instrument
-     * @param null         $visit_label the value of visit lable
-     * @param null         $candidate   the value of candidate
+     * @param string       $visit_label the value of visit lable
+     * @param string       $candidate   the value of candidate
```

Comment:
same

(I'm not sure why these lines are changed in this PR since you're not using them or changing them, but if you're going to fix them they should be fixed right.)